src/app/group/group-add-edit/group-add-edit.component.ts

```diff
@@ -11,68 +12,180 @@ import { filter } from 'rxjs';
 export class GroupAddEditComponent implements OnInit {
 
   addGroupForm !: FormGroup;
-  groupTypeArray: string[] = ['Trip', 'Couple', 'Other'];
+  addMembersForm !: FormGroup;
+
   isGroupAddActive: Boolean = false;
-  getActivatedRouteParam : String = '';
+  getActivatedRouteParam: String = '';
+  isGroupCreated: boolean = false;
+
+  GroupResult !: GroupResult;
+  userProfile!: UserProfile;
 
   // Constructor
-  constructor( private router: Router, private activatedRoute: ActivatedRoute){};
+  constructor(
+    private router: Router,
+    private activatedRoute: ActivatedRoute,
+    private groupsService: GroupsService
+  ) { }
 
   // ngOnInit method
   ngOnInit(): void {
+    /** Form group for add group page */
     this.addGroupForm = new FormGroup({
       groupName: new FormControl('', [Validators.required]),
-      shareLink: new FormControl(''),
-      groupType: new FormControl('Home'),
+      about: new FormControl(''),
       groupMember: new FormArray([
-        this.memberNameAndEmailForm()
+        new FormGroup({
+          userId: new FormControl('3a0ba79a-0f40-0dd1-4e91-9ed501777180')
+        })
       ]),
     })
 
-    // get activatedRoute parameter using observable
-    this.activatedRoute.params.subscribe((param) =>{
+    /** form group add members to group */
+    this.addMembersForm = new FormGroup({
+      groupMembers: new FormArray([
+        this.addMembersToGroupForm()
```

Comment:
Semi colon

Reply:
It showing error when i try to add semi colon here.